types/fundamental-react/lib/Tile/Tile.d.ts

```diff
@@ -1,36 +1,39 @@
+import * as React from "react";
+
 export type TileProps = {
-    active?: boolean,
-    backgroundImage?: string,
-    className?: string,
-    disabled?: boolean,
-    disableStyles?: boolean,
-    productTile?: boolean,
-    tabIndex?: number,
-    onClick?: (...args: any[]) => any
+    active?: boolean;
+    backgroundImage?: string; //TODO does not work yet
+    className?: string;
+    disabled?: boolean;  //TODO does not work yet
+    isDouble?: boolean;
+    size?: "s";
+    onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
 } & { [x: string]: any };
 
-export type TileActionsProps = {
+export type TileContentProps = {
     className?: string;
+    twoColumns?: boolean,
 } & { [x: string]: any };
 
-export type TileContentProps = {
-    title: string,
-    className?: string,
-    headingLevel?: 2 | 3 | 4 | 5 | 6;
-    productTile?: boolean,
-    titleProps?: any
+export type TileFooterProps = {
+    className?: string;
 } & { [x: string]: any };
 
-export type TileMediaProps = {
+export type TileHeaderProps = {
     className?: string;
-    productTile?: boolean;
+    subtitle?: string;
+    //TODO subTitleProps?: https://sap.github.io/fundamental-react/?path=/docs/component-api-tile--primary#properties
+    //TODO titleProps?: 
 } & { [x: string]: any };
 
-declare const Tile: React.FunctionComponent<TileProps> & {
-    displayName: "Tile";
-    Actions: React.FunctionComponent<TileActionsProps> & {displayName: "Tile.Actions"};
-    Content: React.FunctionComponent<TileContentProps> & {displayName: "Tile.Content"};
-    Media: React.FunctionComponent<TileMediaProps> & {displayName: "Tile.Media"};
-};
+
+declare class Tile extends React.Component<TileContentProps> {
```

Comment:
why did your remove the & and made the props static?

Reply:
copied the idea from a component with similar structure, because that worked. 
![image](https://user-images.githubusercontent.com/59808215/98352954-f9fac100-201e-11eb-9a23-2e0526e74806.png)
)

I read up on it and this post says we should favor functional components, if it doesn't need its own state or lifecycle hook. Don't quite understand, will read more about it